### ISSUE TITLE
2012 layout is not as accurate + thumbnail issue

### ISSUE THREAD

**Issue by hastalawindowsvista** [lightbeam24/CustomTube#97]
Okay yeah customtube is cool, but it's not that accurate. Some of the buttons are rounded and use the new font, and ofc, other stuff.
I took a few screenies of the 2012 layout so you can compare them with the CT 2012 theme
![The Evolution of YouTube A Rewind of Layouts - YouTube - 0-3-20](https://github.com/lightbeam24/CustomTube/assets/147812038/94e4962e-7e94-42aa-9dfe-2ddd7bde7548)
![youtube-website^2012^watch-page-gangnam-style](https://github.com/lightbeam24/CustomTube/assets/147812038/481f91c4-5dac-492b-b457-3331259171d5)
Another issue, whenever I hover my mouse over a video's thumbnail that thumbnail suddenly pops up next to the vid's thumbnail

**Comment by lightbeam24** [comment]
Yes, making the 2012 layout more accurate is one of my priorities now. I know what the 2012 layout looks like, the extension as a whole simply isn't finished.

> Another issue, whenever I hover my mouse over a video's thumbnail that thumbnail suddenly pops up next to the vid's thumbnail

Could you send a screen recording of that? I have no clue what you mean.
